app/src/main/res/layout/drawer_list_item.xml

```diff
@@ -0,0 +1,58 @@
+<?xml version="1.0" encoding="utf-8"?>
+<layout xmlns:android="http://schemas.android.com/apk/res/android">
+    <data>
+        <import type="android.view.View"/>
+        <variable
+            name="menu"
+            type="org.fossasia.openevent.app.main.menu.OpevMenu"/>
```

Comment:
It will be beneficial if we make menu class. So that in future we can add attributes to menu item. For example attendeee number or check in checked out number.